README.md

```diff
@@ -21,19 +21,19 @@
 <p align="center">
 	<a href="https://demo.thelounge.chat/"><img
 		alt="#thelounge IRC channel on freenode"
-		src="https://img.shields.io/badge/freenode-%23thelounge-415364.svg?colorA=ff9e18"></a>
+		src="https://img.shields.io/badge/Freenode-%23thelounge-415364.svg?colorA=ff9e18"></a>
```

Comment:
Even freenode itself consistently refers to the network in all-lowercase e.g. in blog posts, press, and so on. It looks weird capitalized because that runs counter to all of their branding.

Reply:
That's a good point, I'm happy to go either way (I personally prefer capitalized, but that's not relevant). The [Wikipedia page of (f|F)reenode](https://en.wikipedia.org/wiki/Freenode) refers to it as "Freenode" while the [Wikipedia page of npm](https://en.wikipedia.org/wiki/Npm_(software)) refers to it as "npm" 🤷‍♂

Reply:
Wikipedia's style guide [says to capitalize trademarks](https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Trademarks#General_rules) in most cases. I wouldn't base any presentation decisions off of the capitalization there. 😅

Reply:
Yeah freenode can stay lowercase, just like npm is.

Reply:
Works for me, done!